domains/berkk.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "berkkaradalan",
+    "email": "berkkaradalan@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://berkkaradalan.github.io/"
```

`CNAME` record must contain a domain name only.
```suggestion
    "CNAME": "berkkaradalan.github.io"
```